docker-compose.yml

```diff
@@ -83,12 +83,25 @@ services:
     volumes:
       - ./docker/elasticsearch5.yml:/usr/share/elasticsearch/config/elasticsearch.yml
 
+  elasticsearch6:
+    image: elasticsearch:6.7.0
+    environment:
+      - http.host=0.0.0.0
+      - transport.host=127.0.0.1
+      - xpack.security.enabled=false
+    healthcheck:
+      << : *default-healthcheck
+      test: "curl --silent --fail localhost:9200/_cluster/health || exit 1"
```

Comment:
Will it be at `localhost:9200`? Sorry I might have misunderstood this.

Reply:
Inside the container it's `localhost:9200`, outside the container it's `elasticsearch6:9200`